lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/remote/impl/DefaultRemoteManagerFactory.java

```diff
@@ -61,22 +64,24 @@ private DefaultRemoteManagerFactory(
     this.tpFactory = tpFactory;
   }
 
-  // TODO[REEF-547]: This method uses deprecated DefaultRemoteManagerImplementation constructor.
   @Override
   public RemoteManager getInstance(final String name) {
-    return new DefaultRemoteManagerImplementation(name,
-        DefaultRemoteManagerImplementation.UNKNOWN_HOST_NAME, // Indicate to use the localAddressProvider
-        0, // Indicate to use the tcpPortProvider
-        this.codec,
-        this.errorHandler,
-        this.orderingGuarantee,
-        this.numberOfTries,
-        this.retryTimeout,
-        this.localAddressProvider,
-        this.tpFactory);
+    try {
```

Comment:
I thought they are legacy. In long term plan, all `getInstance()` might be replaced.